src/Kobo/SystemDialog.cpp

```diff
@@ -46,13 +46,17 @@ class SystemWidget final
     SWITCH_OTG_MODE,
     USB_STORAGE,
     INCREASE_BACKLIGHT_BRIGHTNESS,
-    DECREASE_BACKLIGHT_BRIGHTNESS
+    DECREASE_BACKLIGHT_BRIGHTNESS,
+    INCREASE_BACKLIGHT_COLOUR_TEMPERATURE,
+    DECREASE_BACKLIGHT_COLOUR_TEMPERATURE
```

Comment:
Trailing comma missing

Reply:
Are you sure?  Line 49 (previously last in list) got a comma added.  Line 51 (new last in list) shouldn't have one.

Reply:
Yes, I'm sure. The comma was missing in commit 79b80960035 but this wasn't noticed in code review.

Reply:
Added the comma